domain/src/main/java/org/oppia/android/domain/topic/TopicController.kt

```diff
@@ -912,6 +917,105 @@ class TopicController @Inject constructor(
     }.build()
   }
 
+  private fun getClassroomIdByTopicId(topicId: String): String {
```

Comment:
We priviously discussed injecting ClassroomListController and calling the appropriate fuinctions here.

Reply:
I have moved the function to the `ClassroomController` and added tests for the same.